cocos/platform/winrt/CCWinRTUtils.cpp

```diff
@@ -59,7 +59,6 @@ bool isWindowsPhone()
 #else
     return false;
 #endif
-    return false;
```

Comment:
Glad to. I used a more descriptive title [this time](https://github.com/cocos2d/cocos2d-x/pull/16874).